_posts/2015-01-22-ms-open-tech-china-releases-odata-producer-codegen-for-to-tap-the-gold-mine-of-corporate-data.html

```diff
@@ -12,7 +12,7 @@
 author: Microsoft Open Technologies (China)
 ---
 <p><a href="/assets/JDBC.png"><img class="alignnone wp-image-12808 " src="/assets/JDBC.png" alt="JDBC" width="797" height="446" /></a></p>
-<p>According to a 2013 <a href="https://www.mckinsey.com/insights/business_technology/open_data_unlocking_innovation_and_performance_with_liquid_information" target="_blank">report</a> issued by global consulting firm McKinsey &amp; Company, The flow of “liquid”open data may contribute a total of three to five trillion dollars to the global economy. Committed to promote the free flow of data, Microsoft Open Tech (China) has become an active participant in the effort to build an Open Data Platform within the China market. MS Open Tech (China) not only contributed an <a href="https://www.odata.org/blog/ms-open-tech-china-introduces-azure-based-open-data-solution-using-ckan-and-odata-technologies/" target="_blank">OData plug-in to CKAN</a>, the world’s leading open-source data portal platform to accelerate the China government Open Data initiative, but also to showcase the broader achievements of this effort at the <a href="http://2014.juhe.cn/" target="_blank">Juhe Data Developers Conference</a>, held on Oct 17 in 2014.</p>
+<p>According to a 2013 <a href="https://www.mckinsey.com/insights/business_technology/open_data_unlocking_innovation_and_performance_with_liquid_information" target="_blank">report</a> issued by global consulting firm McKinsey &amp; Company, The flow of “liquid”open data may contribute a total of three to five trillion dollars to the global economy. Committed to promote the free flow of data, Microsoft Open Tech (China) has become an active participant in the effort to build an Open Data Platform within the China market. MS Open Tech (China) not only contributed an <a href="/blog/ms-open-tech-china-introduces-azure-based-open-data-solution-using-ckan-and-odata-technologies/" target="_blank">OData plug-in to CKAN</a>, the world’s leading open-source data portal platform to accelerate the China government Open Data initiative, but also to showcase the broader achievements of this effort at the <a href="http://2014.juhe.cn/" target="_blank">Juhe Data Developers Conference</a>, held on Oct 17 in 2014.</p>
```

Comment:
Broken link https://www.mckinsey.com/insights/business_technology/open_data_unlocking_innovation_and_performance_with_liquid_information

Reply:
@KenitoInc Fixing dead links that point to external resources was beyond the scope of this pull request. In any event, I didn't have a valid alternative I could replace them with.